CI-CD/GitLab/Mend CLI/Gitlab_template/.gitlab-ci.yaml

```diff
@@ -0,0 +1,84 @@
+default:
+  image: maven:3.8-openjdk-11
+
+include:
+  - remote: 'https://raw.githubusercontent.com/mend-toolkit/mend-examples/main/GitLab/Mend%20CLI/Gitlab_template/mend.yml'
+
+  # - project: 'templates/templates'
+  #   file: 'mend.yaml'
+
+# Vars for Maven
+variables:
+  MAVEN_OPTS: -Dmaven.repo.local=$CI_PROJECT_DIR/.m2/repository
+  MAVEN_CLI_OPTS: -DskipTests
+
+stages:
+  - build
+  - download
+  - mend_scan
+  - mend_reports
+
+## Cache exmple for Maven
+cache:
+  key: ${CI_COMMIT_REF_SLUG}
+  paths:
+    - .m2/repository
+    - target
+  policy: pull-push
+
+## Maven example - replace with your builds steps
+# maven_build:
```

Comment:
uncomment and show in readme as shown in https://gitlab.com/samqws-oritest/templates/-/blob/main/README.md